website/content/docs/concepts/seal.mdx

```diff
@@ -157,255 +142,247 @@ and their API equivalents in the [/sys/init](/vault/api-docs/system/init) endpoi
+Additionally, Vault does not initialize if you have not set the option to
+generate a key, and Vault does not find a key. Refer to
 [Configuration](/vault/docs/configuration/seal/pkcs11) for more details.
 
 ### Rekeying
 
 #### Unseal key
 
-Vault's unseal key can be rekeyed using a normal `vault operator rekey`
-operation from the CLI or the matching API calls. The rekey operation is
-authorized by meeting the threshold of recovery keys. After rekeying, the new
-barrier key is wrapped by the HSM or KMS and stored like the previous key; it is not
-returned to the users that submitted their recovery keys.
+You can rekey Vault's unseal key using a normal `vault operator rekey`
+operation from the CLI or the matching API calls. Vault authorizes the rekey operation when you meet the threshold of recovery keys. After rekeying, Vault wraps the new
+barrier key by the HSM or KMS and stores it like the previous key. Vault does not return the barrier key to the users that submitted their recovery keys.
 
 <EnterpriseAlert product="vault">
   Seal wrapping requires Vault Enterprise
 </EnterpriseAlert>
 
 #### Recovery key
 
-The recovery key can be rekeyed to change the number of shares/threshold or to
-target different key holders via different PGP keys. When using the Vault CLI,
-this is performed by using the `-target=recovery` flag to `vault operator rekey`.
+You can rekey the recovery key to change the number of shares/threshold or to
+target different key holders using different PGP keys. When you use the Vault CLI,
+it performs the rekey operation by using the `-target=recovery` flag to `vault operator rekey`.
 
-Via the API, the rekey operation is performed with the same parameters as the
+Using the API, you perform the rekey operation with the same parameters as the
 [normal `/sys/rekey`
 endpoint](/vault/api-docs/system/rekey); however, the
-API prefix for this operation is at `/sys/rekey-recovery-key` rather than
+API prefix for the rekey operation is at `/sys/rekey-recovery-key` rather than
 `/sys/rekey`.
 
 ## Seal migration
 
-The seal migration process cannot be performed without downtime, and due to the
+You cannot perform the seal migration process without downtime. Due to the
 technical underpinnings of the seal implementations, the process requires that
 you briefly take the whole cluster down. While experiencing some downtime may
 be unavoidable, we believe that switching seals is a rare event and that the
 inconvenience of the downtime is an acceptable trade-off.
 
-~> **NOTE**: A backup should be taken before starting seal migration in case
-something goes wrong.
+You should make sure to create a backup before you start the seal migration process, in case something goes wrong.
+
+The seal migration operation requires both the old and new seals to be available during the migration. For example, if you migrate from auto unseal to Shamir seals, the service backing the auto unseal must be available during the migration.
```

Comment:
Removes "Note", as we should be using these sparingly. I think this content makes sense in the context of the procedure and the Note is unnecessary.